modules/custom/alternative_frontpage/src/AlternativeFrontpageInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Drupal\alternative_frontpage;
```

Comment:
Shouldn't this be also in Entity namespace?

Reply:
As just discussed. Nope, this should not be in the entity namespace.